### DISCUSSION TITLE
Swift PM prompts for keychain access in Xcode 14.2

### DISCUSSION THREAD

**Discussion by WiesnerPeti** [sparkle-project/Sparkle#2379] [Q&A]
### Description of the problem
I want to integrate the Sparkle 2 framework through Swift PM per [documentation](https://sparkle-project.org/documentation/).

Each time the project's package versions are resolved, Xcode prompts for accessing the github.com credentials in my login keychain. 
It could be that this is an Xcode related issue, but I successfully integrated ~11 different well-known package without this issue, only Sparkle prompts for this credentials. Developer experience can be solved by choosing "always allow" on the prompt, but my CI script in a Github action runner will also stop and time out on this prompt.

What is different in the Sparkle framework? (might be the url download with checksum in the Package.swift?) Any best practice to overcome this? 

I tried to create a custom build.keychain add the credential there, set it to default, set the search list to prefer this first, unlock it, add xcodebuild/Xcode to the ACL list, but xcodebuild/Xcode still prompts for input. Do you have any pointers?

![Screenshot 2023-05-08 at 16 37 30](https://user-images.githubusercontent.com/4218078/237026105-7aa63a7f-3915-4bcf-a782-a1f4cf28f5bd.png)

### Do you use Sandboxing in your app?
Yes, but I think that is unrelated.

### Version of `Sparkle.framework` in the latest version of your app
Trying to integrate 2.4.1 

### Version of `Sparkle.framework` in the old version of app that your users have (or N/A)
N/A

### Sparkle's output from Console.app
This is for integration, Sparkle is not running yet.

### Steps to reproduce the behavior
1. Xcode 14.2
2. Add a project
3. Follow [documentation](https://sparkle-project.org/documentation/) to add Sparkle to Xcode
4. During the package cloning Xcode should prompt for keychain access.

[The more information provided and pasted verbatim, the easier it will be to diagnose an issue. If you can provide the affected application/binary and XML feed to reproduce an issue, share them]

**Comment by zorgiepoo** [comment]
I don't know. Sparkle vends a binary SPM package if it makes any difference. If this were a real issue in Sparkle or if it was common with Sparkle, more people would have reported this by now. This is the first I've heard of this.
